src/metax_api/api/rpc/base/views/file_rpc.py

```diff
@@ -4,23 +4,34 @@
 #
 # :author: CSC - IT Center for Science Ltd., Espoo Finland <servicedesk@csc.fi>
 # :license: MIT
+
 import logging
 
-from django.db import connection
 from django.conf import settings as django_settings
+from django.db import connection
 from rest_framework.decorators import list_route
 from rest_framework.response import Response
 from rest_framework import status
 
 from metax_api.exceptions import Http400
+from metax_api.services import FileService
 from .common_rpc import CommonRPC
 
-
 _logger = logging.getLogger(__name__)
 
 
 class FileRPC(CommonRPC):
 
+    @list_route(methods=['delete'], url_path="delete_project")
```

Comment:
Although I failed to mention it in the ticket, the http verb should be POST. The rough idea with Metax's RPC API is that GET requests for read operations, and POST for writes (create, updatae, delete). The existing similar "flush_project" also requires POST. Sorry for forgetting that!